testing/environments/snapshot.yml

```diff
@@ -5,7 +5,7 @@ services:
   elasticsearch:
     image: docker.elastic.co/elasticsearch/elasticsearch:8.0.0-SNAPSHOT
     healthcheck:
-      test: ["CMD", "curl", "-f", "http://localhost:9200"]
+      test: ["CMD", "curl", "-f", "http://localhost:9200/_xpack/license"]
```

Comment:
Can you elaborate on this one?

Reply:
This makes it consistent with https://github.com/elastic/beats/blob/6bff9a64b4f18101f62842a9885b2c904ce45c6b/metricbeat/module/elasticsearch/_meta/Dockerfile#L2.